includes/class-wc-siftscience-admin.php

```diff
@@ -287,44 +287,34 @@ private function notice_config() {
 		}
 
 		private function notice_stats() {
-			global $wp;
-			//$sr = $_SERVER['SERVER_NAME'];
-			$sr = $_SERVER['DOCUMENT_ROOT']; 
-			$rwp = 'xxx '.home_url(add_query_arg(array( 'set_siftsci_stats'=> 'yes' ), $wp->request ));
-			$rwp2 = home_url($wp->request);
-			$current_url = home_url($_SERVER['REQUEST_URI']);
-			$hp= get_home_path();
-			
+			$s3k = 'set_siftsci_stats'; // a reusable string
```

Comment:
Good idea to save in a reusable string, but I don't understand what `s3k` means.

Reply:
Also, check out the `WC_SiftScience_Options` class. Maybe this is a better place to put the string constant.

Reply:
set_siftsci_stats 3 wird starts with  S
K there used as a key

Reply:
ah :-). I would prefer a clearer name. How about: `$set_stats_key` ?